plenum/test/consensus/order_service/test_sim_ordering.py

```diff
@@ -7,12 +8,20 @@
     check_batch_count, check_consistency
 from plenum.test.simulation.sim_random import DefaultSimRandom
 
-
 REQUEST_COUNT = 10
 
 
-@pytest.mark.parametrize("seed", range(10))
-def test_ordering_with_real_msgs(seed):
+@pytest.mark.parametrize("seed", range(100))
+def test_ordering_with_real_msgs_default_seed(seed):
```

Comment:
I don't think we really need tests with fixed seeds, when we already have ones with random. Rationale is that when we do have some easily (>5%) reproduced problems it doesn't matter whether seeds are all in 0..100 range or not - only number of them really matters.